### PR TITLE
Upgrade to 5.0~ynh1

### LINKED ISSUE #21: Bullseye install fails : custom apt/deb repo not supporting bullseye
`'http://repo.zabbix.com/zabbix/4.4/debian bullseye Release' does not have a Release file`

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Fix #21*

## Solution

- *Upgrade to 5.0~ynh1*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -159,8 +158,11 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading dependencies..."
 
+	install_zabbix_repo
 	DEBIAN_FRONTEND=noninteractive apt-mark unhold zabbix-server-mysql zabbix-frontend-php
 	ynh_exec_warn_less ynh_install_app_dependencies $pkg_dependencies
+
+	apt-get upgrade -y --option Dpkg::Options::=--force-confdef --option Dpkg::Options::=--force-confold
```
Comment (yalh76) [inline scripts/upgrade:165]: @alexAubin what do you think of that ?
Seems like a bazooka to kill a mosquito, but I didn't find another way to force the upgrade of zabbix from 4.4 to 5.0

Comment (alexAubin) [reply scripts/upgrade:165]: Ah, I think you're referring to the fact that `ynh_install_app_deps` will not upgrade installed dependencies ? We recently added a trick in the apt helpers here (NB: in `install_extra`): https://github.com/YunoHost/yunohost/blob/dev/helpers/apt#L415-L419

Maybe we could do the same for the regular `_install_` helper ... but I'm puzzled as to why this app is not using the `_install_extra` helper ... dependency management for this app seems pretty complex x_x

Comment (yalh76) [reply scripts/upgrade:165]: To do => use ynh_install_extra_app_dependencies

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Pull request merged**